lib.js

```diff
@@ -80,12 +80,12 @@ async function createServer({
 
   const homePage = await fs.readFile(path.resolve(__dirname, 'paintBoard.html'))
     .then((data) => data.toString()
-      .replaceAll('$wsUrl', wsUrl)
-      .replaceAll('$width', width)
-      .replaceAll('$height', height)
-      .replaceAll('$5width', 5 * width)
-      .replaceAll('$5height', 5 * height)
-      .replaceAll('$cd', cd));
+      .replace(new RegExp(/(\$wsUrl)/g), wsUrl)
+      .replace(new RegExp(/(\$width)/g), width)
+      .replace(new RegExp(/(\$height)/g), height)
+      .replace(new RegExp(/(\$5width)/g), 5 * width)
+      .replace(new RegExp(/(\$5height)/g), 5 * height)
+      .replace(new RegExp(/(\$cd)/g), cd));
```

Comment:
```suggestion
      .replace(/\$wsUrl/g, wsUrl)
      .replace(/\$width/g, width)
      .replace(/\$height/g, height)
      .replace(/\$5width/g, 5 * width)
      .replace(/\$5height/g, 5 * height)
      .replace(/\$cd/g, cd));
```
为什么要 `new RegExp()` 呢 :thinking:

Reply:
啊这，可能没区别（